numpy/_core/fromnumeric.py

```diff
@@ -3456,6 +3456,8 @@ def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue, *,
         Alternate output array in which to place the result.  The default
         is ``None``; if provided, it must have the same shape as the
         expected output, but the type will be cast if necessary.
+        Note that the current implementation passes this argument to "reduce-like
+        methods", so the accumulated value should fit in it.
```

Comment:
```suggestion
    See :ref:`ufuncs-output-type` for more details.
```
I think the idea was to link to a central place rather than adding detail to every `out` parameter description.